apps/expo/src/app/index.tsx

```diff
@@ -7,29 +7,31 @@ import { FlashList } from "@shopify/flash-list";
 import type { RouterOutputs } from "~/utils/api";
 import { api } from "~/utils/api";
 
-function PostCard(props: {
+interface PostCardProps {
   post: RouterOutputs["post"]["all"][number];
   onDelete: () => void;
-}) {
+}
+
+function PostCard({ post, onDelete }: PostCardProps) {
```

Comment:
Same, revert destructuring.

Also dont see why the type has been brought out to an interface rather than inlined, as it is inlined everywhwre Else